### ISSUE TITLE
DnCNN training options

### ISSUE THREAD

**Issue by yx-chan131** [cszn/KAIR#83]
Hi, thanks for this great work!
I have some questions regarding DnCNN model training. In the paper, the loss function applied is MSE Loss, may I know why it is stated that L1 loss is preferred in the training JSON?

Besides, I am well aware that there are scripts for the jpeg compression dataset, but how should I do if I want to create a custom dataset for Gaussian noise and jpeg compression?

Thank you very much.
